influence_benchmark/config/accelerate_config.py

```diff
@@ -118,3 +118,57 @@ def to_cli_args(self):
             else:
                 args.append(f"--{k.replace('_', '-')}={v}")
         return args
+
+
+@dataclass
+class AccelerateConfigDeepSpeed(AccelerateConfig):
+    enable_cpu_affinity: bool = False
+    use_cpu: bool = False
+    use_deepspeed: bool = True
+
+    gradient_clipping: float = 15.0
+    offload_param_device: Optional[str] = None
+    offload_optimizer_device: Optional[str] = None
+
+    def set_gpu_ids(self, gpu_ids: Optional[List[int]]):
+        if gpu_ids is None:
+            return
+        self.gpu_ids = gpu_ids
+        self.num_processes = len(self.gpu_ids)
+        print(f"Accelerate training on GPUs: {self.gpu_ids}")
+
+
+@dataclass
+class AccelerateConfigDeepSpeed1(AccelerateConfigDeepSpeed):
+    zero_stage: int = 1
+
+
+@dataclass
+class AccelerateConfigDeepSpeed2(AccelerateConfigDeepSpeed):
+    zero_stage: int = 2
+
+
+@dataclass
+class AccelerateConfigDeepSpeed3(AccelerateConfigDeepSpeed):
+    zero_stage: int = 3
```

Comment:
What are these?

Reply:
These are the different deepspeed configs, the only thing which needs to be different is the stage, i.e. whether to partition optimizer states, params and actiavtions